x/twap/keeper_test.go

```diff
@@ -373,6 +373,33 @@ func newTwapRecordWithDefaults(t time.Time, sp0, accum0, accum1 sdk.Dec) types.T
 	}
 }
 
+func newThreeAssetPoolTwapRecordWithDefaults(t time.Time, sp0, accumA, accumB, accumC sdk.Dec) (types.TwapRecord, types.TwapRecord, types.TwapRecord) {
```

Comment:
Please add godoc specifying how the rest of the spot prices are inferred from `sp0`